packages/material-ui/src/LinearProgress/LinearProgress.js

```diff
@@ -207,10 +208,18 @@ const LinearProgress = React.forwardRef(function LinearProgress(props, ref) {
   const rootProps = {};
   const inlineStyles = { bar1: {}, bar2: {} };
 
+  if (maxValue <= 0) {
+    warning(
+      false,
```

Comment:
```suggestion
      maxValue > 0,
```
Prefer this over wrapping it in if statements